### DISCUSSION TITLE
IP180 compatibility

### DISCUSSION THREAD

**Discussion by cmasse1** [ParadoxAlarmInterface/pai#455] [Q&A]
Hi folks, I've been playing around with PAI for the last couple of days using an EVO 192 board and a IP180 using wifi with a fixed IP address. Has anyone had any success getting the IP180 to talk with PAI - is it even possible?

For reference, I'm treating it in terms of config as if it was an IP150.

**Comment by mremedi2023** [comment]
The same here

**Reply by yozik04** [reply]
@cmasse1 did you got it working?

**Reply by mremedi2023** [reply]
I couldn't, also the old ip150 I had had the firmware updated to a newer version and it doesn't connect anymore either, my evo was left out of my home assistant, sad

**Reply by yozik04** [reply]
Ip150 should work. You are doing something wrong.

**Comment by mremedi2023** [comment]
I didn't change anything in my configuration and before it worked, after updating the firmware of my ip150+ it stopped connecting, now it has 6.10.010

**Comment by mremedi2023** [comment]
![Screenshot_9](https://github.com/ParadoxAlarmInterface/pai/assets/152791674/3197ca0c-88cd-4617-ad34-072bd637863e)

**Reply by mremedi2023** [reply]
![image](https://github.com/ParadoxAlarmInterface/pai/assets/152791674/b498a837-b840-41d2-ae7d-ad62b6d46fb0)

**Reply by yozik04** [reply]
It looks that it does not expose port 10000 anymore. How you connect with Babyware?

**Comment by mremedi2023** [comment]
With serial number and password from the evo, it no longer lets me do it
with a fixed IP

**Reply by yozik04** [reply]
There might be a setting to allow local connections somewhere. Did you check web UI of the IP150?

**Comment by mremedi2023** [comment]
You can no longer enter the web interface on the ip150 with the new
firmware, they do not allow connections to internal networks according to
what I have read, even though I know the IP, you cannot enter through the
browser.

**Comment by cmasse1** [comment]
Hi there - I was able to get the IP180 to connect via InField and BabyWare, but not to PAI. I tried to do a firmware update of the main board with it (Evo 192) but it seems to have failed and bricked it. I'll get a replacement board when I have a bit more spare time to tinker with the system. I do recall connection over IP being listed as a thing for the "future" for the IP180 as per Paradox's document... Although with the IP150 not connecting over IP after a firmware update I am doubtful it will ever happen.

**Reply by yozik04** [reply]
https://github.com/ParadoxAlarmInterface/pai/wiki/Compatibility

I see that EVO fw does not influence. IP150 might.

**Comment by mremedi2023** [comment]
I agree, for now only promises from paradox, the problem is that more and
more they are closing their systems, they are looking more and more like
Apple

**Comment by mremedi2023** [comment]
Yesterday I was looking at that table that is very useful. I got an old
ip150, it arrives in a few days, I'll try that one and see what happens. I
have already closed all the ports on the router where it will supposedly
seek to update automatically.

**Comment by DrMaboul** [comment]
IP180 work with a new protocol from PARADOX, based on MQTT (probably proprietary, I don't own any MQTT device from PARADOX).
Its little brother IP150MQ work also like this (supplied with firmware v6.x)

For other IP150's, stay away from firmware jump to 6.x as you'll loose direct IP connection and, at this stage, compatibility with PAI.

**Comment by mremedi2023** [comment]
After updating to its latest firmware, ip150+ became a 150MQ, I understand
that it is now MQTT, but everything does not connect to the panel, except
that something else has to be modified in the PAI configuration and I don't
know it.


[image: image.png]

**Reply by DrMaboul** [reply]
For now, only way would be to go back to firmware v5.x

**Option 1** : maybe is it possible to downgrade to 5.x. Do you see any different firmware version in InField ? It seems that with BabyWare 5.6.40, the included In-Field could offer a wayback "prod/IP150+5.6.14TURN.PUF". Do you see this option ?

It's the case with people who upgrade their PCS265 from v7 to v8 (here v8 is the MQTT version). Fall-back to v7 is possible and mentioned in the installation manual. With a little luck, Paradox also previewed choice mistake for the IP150.

**Option 2** : (from IP150 installation manual) _Firmware Version Fallback : to revert the IP150MQ module to its previously installed firmware version, unplug the power cable from the panel and insert a pin/straightened paper clip (or similar object) into the pinhole located between the two I/O LEDs. Press down gently until you feel some resistance; plug in the power cable while holding the pin down for approximately five seconds and release it when the I/O 2 LED starts flashing (figure 2). The I/O LED will turn solid and start blinking (resetting to the backup version). Once complete, the IP150MQ will reboot automatically to the previously installed firmware version._

**Reply by yozik04** [reply]
But why would you need PAI if you have an official MQTT solution? Is it paid? Can you configure your own MQTT broker in the IP module?

**Reply by DrMaboul** [reply]
> But why would you need PAI if you have an official MQTT solution? Is it paid? Can you configure your own MQTT broker in the IP module?

I don't own one myself and a this point don't really want to upgrade an IP150 to v6 MQTT but as far as I've read until now, it's based on MQTT protocol and use the default SSL/TLS over MQTT port 8883 but you can't really edit a lot of thing apart from the IP but not sure it's so simple.

Just curious but not enough yet to "maybe" sacrifice an IP150.

Maybe if @mremedi2023 can go back to 5.x

**Comment by DrMaboul** [comment]
With firmware 6.x+, you don't have direct IP access like before. As for Babyware like you already noticed. So it shouldn't work with PAI.

Maybe in "swan cloud" mode which use serial but need a site ID and an email (look at the wiki). I never tested this mode with PAI.
It means that you have to register you panel through the Paradox cloud and hope "swan" mode is still compatible following Paradox migration to Insite Gold, then Blue Eye or whatever is the new cloud app for MQTT.

**Comment by mremedi2023** [comment]
It does not work, only for swan, according to the paradox site "Does not support static IP connection for now (should be supported in future firmware updates)" they are closing the possibility of connecting with third party software (in this case home assistant) for everyone On the other hand, I got an old ip150 with a fixed IP and I was able to connect, but I couldn't do it to the blueeye as the firmware was so old, the minimum to be able to use the blueeye is V4.10.000, I updated it to test and if I connect to the app but no more to the home assistant, I have 3 different evo panels and 1 ip150, 1 ip150+ and 1 ip180 and none of them are compatible between combinations of panels + ip modules that I have, according to the compatibility table of the PAI project https://github.com/ParadoxAlarmInterface/pai/wiki/Compatibility a crazy thing... to complete, I cannot put the desired firmware on any IP module, since when you connect to the infield it does not allow you to do anything other than take it to the latest firmware available, obviously they also removed the possibility of doing any downgrade class neither to the modules nor to the IP.

**Reply by DrMaboul** [reply]
Firmware v4.x is real crap, only SWAN.

Firmware v1.30 to 1.40 (IP150 and IP150S) are old but fully opened (ip direct access, web interface, work with PAI)
In the v5.x series, some have web UI and IP direct access not all. I think I have a list somewhere with which one allow not only Paradox cloud.

Did you tried this : 

> Option 2 : (from IP150 installation manual) Firmware Version Fallback : to revert the IP150MQ module to its previously installed firmware version, unplug the power cable from the panel and insert a pin/straightened paper clip (or similar object) into the pinhole located between the two I/O LEDs. Press down gently until you feel some resistance; plug in the power cable while holding the pin down for approximately five seconds and release it when the I/O 2 LED starts flashing (figure 2). The I/O LED will turn solid and start blinking (resetting to the backup version). Once complete, the IP150MQ will reboot automatically to the previously installed firmware version.

What are your panels and firmware version of IPxxx ?

**Comment by mremedi2023** [comment]
Hello, I'm going to try it and then I'll buy the experience, the tested
compatibility table is here:
https://github.com/ParadoxAlarmInterface/pai/wiki/Compatibility

I have 3 panels

Evo192: v.7.70.001
Evo192: v 2.20.001
EvoHD: v.1.06.001

ip150: v.4.10.000
ip150+ (was modified to 150mq after an update): v.6.10.010
ip180: v.1.00.005

**Comment by mremedi2023** [comment]
Option 2: not working

**Reply by yozik04** [reply]
I'd update EVO192 and HD panels to the last 6.x version. You could use Serial connection for these If you do not have an IP150 with supported local connection.

**Comment by mremedi2023** [comment]
For those more knowledgeable in programming, now blue eye has information that it did not have before, I attach a screenshot. Additionally, ip180 allows fixed IP in its latest version, I still cannot connect through the normal PAI procedure.
![image](https://github.com/user-attachments/assets/9057ee51-d759-4b05-aa0b-c0b945014516)

**Reply by yozik04** [reply]
Doesn't it allow you to set your own mqtt broker address? What does it send there?

**Reply by mremedi2023** [reply]
I really don't know, I wouldn't know what to change or put in each place to test. I gave up on making it work, I'm about to change my alarm to a Hikvision one.

**Comment by mremedi2023** [comment]
Hi, in the compatibility table regarding evo hd+ I assume that when it comes to version 1.3.1 or 1.4.1 it actually refers to [ V1.03.001] and [ V1.04.001] is that correct?

**Reply by yozik04** [reply]
Most probably.

**Comment by mremedi2023** [comment]
Sorry but I can't edit the compatibility table, the last 3 lines should be
like this to be correct:


[image: image.png]

I was able to connect my Evohd+ with the latest firmware but I don't know
what happened to my old ip150+ but Paradox took pity and it seems that it
now allows firmware downgrade of the modules, at least mine.

**Reply by yozik04** [reply]
No image was attached.

**Comment by mremedi2023** [comment]
![image](https://github.com/user-attachments/assets/2bc77ff9-d7a4-4f53-b318-b832b6ad85ad)

**Reply by swazz99jhb** [reply]
For an IP180 connected to a EVO 192, Can one put your own MQTT Broker address in the "MQTT development server" section and thus get it to work with HA/Openhab?

**Reply by yozik04** [reply]
Would be interesting if somebody with IP180 would try and tell us. Then PAI will be obsolete.

**Reply by rob196** [reply]
I have an IP180, I just don't have an Installer ID to access the required setting(s) - anyone know how I can get one?

**Comment by BossNeo** [comment]
any update to compatibility with ip180 i have from 3 weeks but is inposible to connect to babywire in any way. Strange is that my payment for blueye is still 0 dolars and -1 day expired. If i need to change it ip150Plus is still the best choise ?

**Reply by yozik04** [reply]
No updates unfortunately.

**Comment by aramsina** [comment]
I’m another victim of IP150+ firmware upgrade to 6.xx.
Like others, I can’t get my panel connected to the PAI.

I noticed when using Babyware to connect to the panel, there is also another required parameter called “System Login / Panel ID” besides the “System / PC Password”.

I have to provide both parameters to successfully connect to my panel. And l can’t see anywhere in the yaml config to set this parameter. Am I missing something or maybe by adding this parameter to the config everyone can get connected again?

MG5000 Version: 04.84.04
IP150+: Version: 6.01.004

Screenshot attached.
Thanks in advance.
![image](https://github.com/user-attachments/assets/8d538706-a7aa-41c8-a503-9b48cad3da1c)

**Reply by yozik04** [reply]
System login info is added automatically in PAI. So it is still there in the connection request.
It is strange that you can connect with babyware, but can't with PAI.
What error is in the PAI logs?
If it is about Password try replacing '0' with 'a' in PC Password. Maybe it will help.

**Reply by aramsina** [reply]
Thanks for your quick response Jevgeni,

So you mean the “System Login / Panel ID” will be sent automatically to SWAN server? As I know it is a user defined value and it’s different from Panel Serial number. Let’s say I have set it to “D123” and set the “System / PC Password” to “B123”. So “D123” is not needed to be defined in yaml file?

Anyhow, these are the logs and my config file:





LOGGING_LEVEL_CONSOLE: 20
LOGGING_LEVEL_FILE: 40
CONNECTION_TYPE: IP
SERIAL_PORT: /dev/ttyUSB0
SERIAL_BAUD: 9600
IP_CONNECTION_HOST: 127.0.0.1
IP_CONNECTION_PORT: 10000
IP_CONNECTION_PASSWORD: paradox
KEEP_ALIVE_INTERVAL: 10
IO_TIMEOUT: 0.5
LIMITS:
  zone: auto
  user: 1-10
  door: ""
  pgm: 1-5
  partition: auto
  module: ""
  repeater: ""
  keypad: ""
  key-switch: ""
SYNC_TIME: true
SYNC_TIME_MIN_DRIFT: 120
PASSWORD: B123
MQTT_ENABLE: true
MQTT_HOST: core-mosquitto
MQTT_PORT: 1883
MQTT_KEEPALIVE: 60
MQTT_USERNAME: paradox
MQTT_PASSWORD: paradox
MQTT_HOMEASSISTANT_AUTODISCOVERY_ENABLE: true
COMMAND_ALIAS:
  arm: partition all arm
  disarm: partition all disarm
MQTT_COMMAND_ALIAS:
  armed_home: arm_stay
  armed_night: arm_sleep
  armed_away: arm
  disarmed: disarm
HOMEASSISTANT_NOTIFICATIONS_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
HOMEASSISTANT_PUBLISH_PARTITION_PROPERTIES:
  - target_state
  - current_state
HOMEASSISTANT_PUBLISH_ZONE_PROPERTIES:
  - open
  - tamper
PUSHBULLET_CONTACTS: []
PUSHBULLET_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
PUSHOVER_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
PUSHOVER_BROADCAST_KEYS: []
SIGNAL_CONTACTS: []
SIGNAL_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
GSM_CONTACTS: []
GSM_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
IP_INTERFACE_ENABLE: false
IP_INTERFACE_PASSWORD: paradox
DUMMY_EVENT_FILTERS: []
IP_CONNECTION_SITEID: t*******t
IP_CONNECTION_EMAIL: t*******t@gmail.com




Attempting to load configuration from /data/options.json
2025-06-13 00:17:34,956 - INFO     - PAI - Starting Paradox Alarm Interface 3.6.1
2025-06-13 00:17:34,957 - INFO     - PAI - Config loaded from /data/options.json
2025-06-13 00:17:34,957 - INFO     - PAI - Console Log level set to 20
2025-06-13 00:17:35,051 - INFO     - PAI.paradox.interfaces - Starting BasicMQTTInterface Interface
2025-06-13 00:17:35,053 - INFO     - PAI.paradox.interfaces.mqtt.core - MQTT loop started
2025-06-13 00:17:35,063 - INFO     - PAI.paradox.interfaces - Starting HomeAssistantMQTTInterface Interface
2025-06-13 00:17:35,064 - INFO     - PAI - Starting...
2025-06-13 00:17:35,066 - INFO     - PAI.paradox.paradox - Connecting to interface
2025-06-13 00:17:35,067 - INFO     - PAI.paradox.paradox - Using IP Connection
2025-06-13 00:17:35,458 - INFO     - PAI.paradox.connections.ip.connection - Connecting. Try 1/3
2025-06-13 00:17:35,459 - INFO     - PAI.paradox.connections.ip.stun_session - Connecting to Site: t*******t
2025-06-13 00:17:35,460 - INFO     - PAI.paradox.connections.ip.stun_session - Getting site info
2025-06-13 00:17:36,374 - INFO     - PAI.paradox.interfaces.mqtt.core - MQTT Broker Connected
2025-06-13 00:17:36,694 - WARNING  - PAI.paradox.connections.ip.stun_session - Unable to get site info. Retrying...
2025-06-13 00:17:44,236 - ERROR    - PAI.paradox.connections.ip.connection - Unhandled exception while connecting (try 1/3): STUN Connect Request error: {'class': b'\x00\x00\x04', 'code': 0, 'reason': b'Bad Peer Address'}
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/connection.py", line 28, in connect
    await self._try_connect()
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/connection.py", line 150, in _try_connect
    await self.stun_session.connect()
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/stun_session.py", line 50, in connect
    stun_r = await self._stun_connect(xoraddr)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/stun_session.py", line 73, in _stun_connect
    raise ConnectToSiteFailed(
paradox.exceptions.ConnectToSiteFailed: STUN Connect Request error: {'class': b'\x00\x00\x04', 'code': 0, 'reason': b'Bad Peer Address'}
2025-06-13 00:17:44,249 - INFO     - PAI.paradox.connections.ip.connection - Connecting. Try 2/3
2025-06-13 00:17:44,249 - INFO     - PAI.paradox.connections.ip.stun_session - Connecting to Site: t*******t
2025-06-13 00:17:45,653 - ERROR    - PAI.paradox.connections.ip.connection - Unhandled exception while connecting (try 2/3): STUN Connect Request error: {'class': b'\x00\x00\x04', 'code': 0, 'reason': b'Bad Peer Address'}
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/connection.py", line 28, in connect
    await self._try_connect()
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/connection.py", line 150, in _try_connect
    await self.stun_session.connect()
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/stun_session.py", line 50, in connect
    stun_r = await self._stun_connect(xoraddr)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/stun_session.py", line 73, in _stun_connect
    raise ConnectToSiteFailed(
paradox.exceptions.ConnectToSiteFailed: STUN Connect Request error: {'class': b'\x00\x00\x04', 'code': 0, 'reason': b'Bad Peer Address'}
2025-06-13 00:17:45,657 - INFO     - PAI.paradox.connections.ip.connection - Connecting. Try 3/3
2025-06-13 00:17:45,657 - INFO     - PAI.paradox.connections.ip.stun_session - Connecting to Site: t*******t
2025-06-13 00:17:47,238 - ERROR    - PAI.paradox.connections.ip.connection - Unhandled exception while connecting (try 3/3): STUN Connect Request error: {'class': b'\x00\x00\x04', 'code': 0, 'reason': b'Bad Peer Address'}
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/connection.py", line 28, in connect
    await self._try_connect()
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/connection.py", line 150, in _try_connect
    await self.stun_session.connect()
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/stun_session.py", line 50, in connect
    stun_r = await self._stun_connect(xoraddr)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/paradox/connections/ip/stun_session.py", line 73, in _stun_connect
    raise ConnectToSiteFailed(
paradox.exceptions.ConnectToSiteFailed: STUN Connect Request error: {'class': b'\x00\x00\x04', 'code': 0, 'reason': b'Bad Peer Address'}
2025-06-13 00:17:47,241 - ERROR    - PAI.paradox.paradox - Failed to connect to interface
2025-06-13 00:17:47,241 - ERROR    - PAI - Unable to connect to alarm
2025-06-13 00:17:50,246 - INFO     - PAI - Starting...
2025-06-13 00:17:50,246 - INFO     - PAI.paradox.paradox - Disconnecting from the Alarm Panel
2025-06-13 00:17:50,247 - INFO     - PAI.paradox.paradox - Clean Session
2025-06-13 00:17:50,248 - INFO     - PAI.paradox.paradox - Connecting to interface
2025-06-13 00:17:50,248 - INFO     - PAI.paradox.connections.ip.connection - Connecting. Try 1/3
2025-06-13 00:17:50,249 - INFO     - PAI.paradox.connections.ip.stun_session - Connecting to Site: t*******t

**Reply by aramsina** [reply]
I replaced the site id and email with asterisks but it's not showing in the text I sent here.

The values are in this format as the screenshot:
![image](https://github.com/user-attachments/assets/fc4123e9-f4b6-4374-86f6-743b8aff78ed)


And also I don't have any 0 in my PC Password.

**Reply by yozik04** [reply]
Can you make local connection without SWAN? Directly to IP module ip?

**Reply by aramsina** [reply]
No, not anymore after upgrading to version 6.
Locally IP150+MQ (former IP150+) only responds to ping, that's all!

**Reply by yozik04** [reply]
Ohh. Bad luck then. We do not support MQTT for connection to the alarm. Maybe someone will develop it in future. Were you forced to update? Can you roll it back to 5.x?

**Reply by aramsina** [reply]
No I wasn't forced.
Actually at the time I was planning to integrate it with my Home Assistant, but never started to do it. Randomly one day opened the BlueEye app and saw the update available to MQTT and said to myself "that's it! Now it can be integrated easily!" And now where I am.

Thank you anyway for following up my issue. I really appreciate it.

**Reply by aramsina** [reply]
I didn't find a way to downgrade it.
There used to be a a drop down menu in the InField app that let the user to select to desired firmware from a list or even select the file from his computer. Now that option is completely gone. It only allows to UPGRADE to the latest version.

That would be great if someone could find a way to downgrade the module.

**Reply by yozik04** [reply]
It is possible that you will need an older version of InField to do the downgrade. I am not sure how safe it is. I never upgraded since some latest 1.x. Works perfectly.

**Comment by ArnoldMo** [comment]
Good day,
For IP180  via blue eye for I can set a static IP,
MQQT broker can only be changed to:
MQTT Production Server
mqtt-ip.insightgoldatpmh.com

MQTT Development Server
mgtt-ip.sa.insightgoldatpmh.com

via BabyWare I can connect via:
Panel S/N
Site ID
Site Token
I would like to try the IPC10 but I think pai those not support it? 
<img width="281" height="372" alt="image" src="https://github.com/user-attachments/assets/7487fc7d-97a4-4155-b287-f62755e22b93" />

EVOHD+ v1.09.000
IP180 v1.00.025

**Reply by yozik04** [reply]
Interesting if it is regular MQTT. Maybe it is possible to subscribe to your own panel's topics somehow.

**Reply by ArnoldMo** [reply]
I can chose between
mqtt-ip.insightgoldatpmh.com
mgtt-ip.sa.insightgoldatpmh.com

could add a local dns
I do not have much expirience with mqtt but I do not know what logins it uses and it would probably have a tls certificate that it trusts
Thats why I am thibking of IPC10
